ansible/roles/redis/tasks/deploy.yml

```diff
@@ -21,10 +21,13 @@
       - "{{ redis.port }}:6379"
     env:
       TZ: "{{ docker.timezone }}"
+    command:
+      /bin/sh -c
+      "docker-entrypoint.sh --requirepass {{ redis.password }}"
 
 - name: wait until redis is up and running
-  action: shell (echo PING; sleep 1) | nc {{ ansible_host }} {{ redis.port }}
+  action: shell (printf "*2\r\n\$4\r\nAUTH\r\n\${{redis.password|length}}\r\n{{ redis.password }}\r\n*1\r\n\$4\r\nPING\r\n"; sleep 1) | nc {{ ansible_host }} {{ redis.port }}
```

Comment:
@kpavel - can you provide a sample `printf` output that this line generates?

Reply:
Talking raw redis is a little bit cryptic. Example printf for password "openwhisk": printf "*2\r\n\$4\r\nAUTH\r\n\$9\r\nopenwhisk\r\n*1\r\n\$4\r\nPING\r\n"
*2
$4
AUTH
$9
openwhisk
*1
$4
PING